server/src/main/java/fr/uga/l3miage/tp4/models/ExamEntity.java

```diff
@@ -0,0 +1,25 @@
+package fr.uga.l3miage.tp4.models;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Entity
+public class ExamEntity {
+    @Id
+    private Long id;
+    private LocalDateTime startDate;
+    private LocalDateTime endDate;
+    @Column(unique = true)
+    private String name;
+    private Integer weight;
+    @OneToMany(mappedBy = "examEntity")
+    private Set<CandidateEvaluationGridEntity> candidateEvaluationGridEntities;
+
+    @ManyToMany
+    @JoinColumn(name = "id_skill", referencedColumnName = "id")
+    private Set<SkillEntity> skillEntities;
```

Comment:
idem ici pas `@JoinColumn` car en @ManyToMany
```suggestion
    @ManyToMany
    private Set<SkillEntity> skillEntities;
```